clients/linux/src/filetree.rs

```diff
@@ -183,38 +192,49 @@ impl FileTree {
                         parent = model.get_iter(&path).unwrap();
                     }
                     _ => {
-                        if tree_iter_value!(model, &parent, 3, String) == format!("{:?}", FileType::Document)  {
+                        if tree_iter_value!(model, &parent, 3, String) == format!("{:?}", FileType::Document) {
                             path.up();
                             parent = model.get_iter(&path).unwrap();
                         }
                     }
                 }
```

Comment:
I know this entirety is not part of this PR right now, but can this match statement not just be an if statement with those conditions (`pos == Before || pos == After) || tree_iter_value!(model, &parent, 3, String) == format!("{:?}", FileType::Document)`)?